### PR TITLE
Adds Admin login & fixes to Chat

### PR DESCRIPTION
**Pull request by ASimpson87**
Adds an admin login through chat rather then having someone to do it through console. Also fixes to the layout.

### CODE REVIEW

File: NitroxServer/ConsoleCommands/LoginCommand.cs [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            {
+                if (playerData.UpdatePlayerPermissions(playerName, Perms.ADMIN))
+                {
+                    message = "Updated permissions to admin";
+                }
+                else
+                {
+                    message = "Could not update permissions " + player.ToString();
+                }
+            }else
```
Comment (Measurity) [inline NitroxServer/ConsoleCommands/LoginCommand.cs]: Format the document to conform to code style.

File: NitroxServer/ConsoleCommands/LoginCommand.cs [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        {
+            string pass = args[0];
+            string message;
+            string playerName = player.Get().Name;
+
+            if (pass == serverConfig.ServerAdminPassword)
+            {
+                if (playerData.UpdatePlayerPermissions(playerName, Perms.ADMIN))
+                {
+                    message = "Updated permissions to admin";
```
Comment (Measurity) [inline NitroxServer/ConsoleCommands/LoginCommand.cs]: Would be good to add the playername that got updated so server owner knows what's happening.

File: NitroxServer/ConsoleCommands/LoginCommand.cs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+                }
+                else
+                {
+                    message = "Could not update permissions " + player.ToString();
```
Comment (Measurity) [inline NitroxServer/ConsoleCommands/LoginCommand.cs]: Add playername here as well.

**Comment by Measurity** [comment]
The default password can be a security vulnerability if the server owner does not know this login command exists. Can you add a warning to the server startup so that server owners can do something about it? If you do this I also recommend adding a command to easily set the password (or any other server setting) at runtime and save it.

Maybe the latter is a bit too much work for 1 PR. Just adding the message to server console is good enough too.

**Comment by Sunrunner37** [comment]
Maybe randomly generate a password (and spit to the console) if a default one is not set

**Comment by ASimpson87** [comment]
Password is now randomly generated if there is nothing set in the config. You can also update the password at run-time with /changepassword [args]

**Pull request merged**